lib/membrane_element_rawvideo/parser.ex

```diff
@@ -88,6 +88,12 @@ defmodule Membrane.Element.RawVideo.Parser do
       {:ok, %{state | queue: payload}}
     else
       {bufs, tail} = split_into_buffers(payload, frame_size)
+
+      {bufs, state} =
+        Enum.map_reduce(bufs, state, fn buffer, state_acc ->
+          {%Buffer{buffer | metadata: %{pts: state_acc.timestamp}}, bump_timestamp(state_acc)}
+        end)
+
```

Comment:
This should calculate timestamps only if they aren't already present in buffers

Reply:
In general, yes, why not? Here I'm unable to imagine a use case, but if it happened, it would be really hard to figure out that something is overriding them. It would be better to raise in such case.